Form-Controls/index.html

```diff
@@ -13,15 +13,47 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+  <form id="orderForm" onsubmit="return validateForm()">
+    <label for="name">Full Name:</label>
+    <input type="text" id="name" name="name">
+    <div id="nameError" class="error"></div>
+
+    <label for="email">Email Address:</label>
+    <input type="email" id="email" name="email">
```

Comment:
What edge cases there could be? Does a 0 character entry valid?

Reply:
here I added require and pattern as well to validate the email so if someone enter 0 the massage says please inter valid input